__tests__/app.test.js

```diff
@@ -106,3 +106,40 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200 - respond with an array of comments for the given article_id ordered by the most recent comment first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) =>
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: 1,
+          })
+        );
+
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
```

Comment:
You could add in a test for a status 200, valid ID, but has no comments and responds with an empty array of comments 💡

Reply:
This was added in PR https://github.com/dku7/be-nc-news/pull/8